core/integration/features/spot/orders/0080-SPOT-024.feature

```diff
@@ -98,4 +98,29 @@ Feature: Spot market bug (from the incentive) replication
     Then "party3" should have general account balance of "1000" for asset "USD"
     Then "party4" should have general account balance of "1" for asset "BTC"
 
+    And the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference | error |
+      | party3 | BTC/USD   | buy  | 20     | 10    | 0                | TYPE_LIMIT | TIF_GTC | p3-b2     |       |
+
+    And the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference | error |
+      | party4 | BTC/USD   | sell | 1      | 66000 | 0                | TYPE_LIMIT | TIF_GTC | p4-s2     |       |
+
+    #0080-SPOT-026: amend order - order is amended such that would trade immediately and the party can't afford none/some of the trades
+    When the parties amend the following orders:
+      | party  | reference | price | size delta | tif     | error                                                              |
+      | party3 | p3-b2     | 66000 | -10        | TIF_GTC | party does not have sufficient balance to cover the trade and fees |
+      | party4 | p4-s2     | 59000 | 2          | TIF_GTC | party does not have sufficient balance to cover the new size       |
+
+    #0080-SPOT-027:submit order - limit order, partly matched, party can't afford the trades
```

Comment:
if it partly matches how come there's no trade at this point and only the remaining of the order get cancelled?

Reply:
party3 does not have sufficient balance to cover the trade

Reply:
I have added more just now, party5 has 70,000 USD to start with, placed a buy order of size 2, price 61000 (so party5 could cover 1 but can not cover the remaining 1)

and no trade is happening